### PR TITLE
CTPPS Pixel DQM: add 1 histogram

### PR DESCRIPTION
**Pull request by popovvp**
add new plot rpot occupancy per BX

### CODE REVIEW

**Comment by threus** [comment]
@popovvp please fix the conflicts if you want to have this PR in 921 (to be used in online DQM). thanks

**Comment by dmitrijus** [comment]
+1

File: DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc [hunk 1]
```diff
@@ -337,22 +341,16 @@ void CTPPSPixelDQMSource::beginLuminosityBlock(edm::LuminosityBlock const& lumiS
...
-  for(int ind=0; ind<2*3*NRPotsMAX*NplaneMAX; ind++)  {
-    for(int rp=0; rp<NROCsMAX; rp++) {
-      HitsMultROC[ind][rp] = 0;
-    }
-  }
+ int RPactivity[NArms][NRPotsMAX], digiSize[NArms][NRPotsMAX];
+ for(int arm = 0; arm <2; arm++) for(int rp=0; rp<NRPotsMAX; rp++)
+   RPactivity[arm][rp] = digiSize[arm][rp] = 0;
+ 
+ for(int ind=0; ind<2*3*NRPotsMAX; ind++)
```
Comment (davidlange6) [inline DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc]: @popovvp - better:
int RPactivity[NArms][NRPotsMAX]={0}, digiSize[NArms][NRPotsMAX]={0};

**Comment by davidlange6** [comment]
-1 
pending answers to small comment
hi @popovvp - in addition this PR needs to be rebased

**Comment by vanbesien** [comment]
Ping!
We have just removed this PR from the online, since it doesn't merge anymore and otherwise blocks us from moving forward for any of the other sub-systems.
Alerting CT-PPS conveners.

File: DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc (hunk 1 above)
Comment (popovvp) [inline DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc]: that is initialization during compiling stage, but I need
 initialization in every analyzed event...
 or this assumption is wrong ?
 thanks,
   Vladimir
for today this PR is already obsolete (overridden by PR 19670/19895)

**Comment by popovvp** [comment]
Hi Broen,

  this PR is oblosete, instead the PR 19895 overrides it for 92x release.
  PR 18835 can be closed.

  thanks,
    Vladimir

**Comment by vanbesien** [comment]
In that case, please close it.
The creator has the close button.

**Comment by popovvp** [comment]
closed due to override by PRs 19670/19895

**Pull request closed without merging**